rune/libenclave/internal/runtime/pal/skeleton/sgxsign.c

```diff
@@ -539,6 +541,12 @@ int main(int argc, char **argv)
 		case 's':
 			meta_data.max_mmap_size = atoi(optarg);
 			break;
+		case 'x':
+			meta_data.xfrm = atoi(optarg);
```

Comment:
Use strtol() to convert the input hex string to integer value instead. Typing a decimal number as input is counter-intuitive.